source/lib/index.ts

```diff
@@ -56,10 +60,10 @@ export default async (options: Options = {cwd: process.cwd()}) => {
 	}
 
 	const pkg = pkgResult.packageJson;
-
 	const config = loadConfig(pkg as any, options.cwd);
 
-	// Look for a typings file, otherwise use `index.d.ts` in the root directory. If the file is not found, throw an error.
+	// Look for a typings file if not explicitly specified, otherwise use `index.d.ts` in the root directory.
```

Comment:
The comment has changed but the implementation hasn't. Was the old comment incorrect?